.travis.yml

```diff
@@ -20,6 +20,7 @@ addons:
       - zlib1g-dev
       - libgtk2.0-dev
       - libexpat1-dev
+      - nodejs
```

Comment:
It actually might be better to change the `language` at the beginning of the travis file, rather than install node for each run: indeed, currently there is no builtin support for ocaml as a language in travis, so the language used is C because it's the most lightweight, but now that we test js, we might as well use its native support.

Reply:
Ok, i hope this solution does not add too much weight in the CI